src/utils/hooks/useReassignReviewAssignment.ts

```diff
@@ -62,28 +64,66 @@ const useReasignReviewAssignment: UseReassignReviewAssignment = (structure) => {
       templateElementId: element.element.id,
     }))
 
+    const unassignedSectionCodes: string[] =
```

Comment:
I'm not sure if this would work... For example if you have 3 sections, and 2 of them were already assigned to Reviewer1, now you are re-assigning Section 2 (but section 3 is still not-assigned to anyone). This would return as Sections 1 AND 3 are part of the `unassignedSectionCodes` which is not accurate. Also the variable naming is confusing, it seems as you are **unassigning** this sections instead of saying this are the **still assigned** sections for the unassignment

Reply:
I think I see what you're saying, but I think it's might actually just be a terminology problem. Perhaps this should just be re-named -- "sectionsNotAssigned" or something? It's basically ALL the sections not currently assigned to this reviewer, not ones we are specifically un-assigning. Is that right?